doc/mgr/prometheus.rst

```diff
@@ -152,6 +153,13 @@ The metrics take the following form:
     ceph_health_detail{name="OSD_DOWN",severity="HEALTH_WARN"} 1.0
     ceph_health_detail{name="PG_DEGRADED",severity="HEALTH_WARN"} 1.0
 
+The module also maintains an in-memory history of health check stats
+By default, it retains a maximum of 1000 entries, These limits are configurable via the following runtime option:
+
+  ``mgr/prometheus/healthcheck_history_max_entries`` - the maximum number of unique health check to track in memory (default: 1000).
```

Comment:
Should have plural here I think so I added "entries" that's consistent with the previous text.

```suggestion
  ``mgr/prometheus/healthcheck_history_max_entries`` - the maximum number of unique health check entries to track in memory (default: 1000).
```